### DISCUSSION TITLE
Do Duckdb WASM Supports Angular?

### DISCUSSION THREAD

**Discussion by akash-goel** [duckdb/duckdb-wasm#1184] [General]
Hi Team,

I am working on Angular Project and want to use WASM , Request you to guide do duckdb supports Angular and Is there any sample project in Angular for Duckdb.

Regards,
AKash

**Comment by sswatson** [comment]
It's a bit of an indirect answer, but: the way you use a library like DuckDB WASM is pretty orthogonal to your choice of frontend framework. You just want to run the initialization code supplied in the DuckDB README in whatever manner is appropriate for your frontend framework (`useEffect` in React, `onMount` in Svelte, maybe `ngOnInit` in Angular -- I don't know Angular).
